api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -127,6 +126,22 @@ public Workspace get(String ns, String firecloudName) {
                 fcWorkspace -> fcWorkspace));
   }
 
+  @Override
+  public Map<User, WorkspaceAccessEntry> getFirecloudWorkspaceAcls(Workspace workspace) {
+    WorkspaceACL firecloudWorkspaceAcls =
+        fireCloudService.getWorkspaceAcl(
```

Comment:
Opt: this is the only place we call getWorkspaceAcl, could the logic after this fold into `fireCloudService.getWorkspaceAcl` and then have the firecloudService method return the data structure we want?